src/datasource/ZappPipesService.js

```diff
@@ -1,4 +1,4 @@
-import ZappPipes from './ZappPipes';
+import APZappPipes from './ZappPipes';
```

Comment:
just for you to know, when importing without brackets on a file with `export default` you can name the module any way you want. so you could leave the `ZappPipes` name here if you want

Reply:
thanks